az-core/src/main/java/azkaban/Constants.java

```diff
@@ -109,13 +109,13 @@
     // Designates one of the external link topics to correspond to a job log viewer
     public static final String AZKABAN_SERVER_EXTERNAL_LOGVIEWER_TOPIC = "azkaban.server.external.logviewer.topic";
     public static final String AZKABAN_SERVER_EXTERNAL_LOGVIEWER_LABEL = "azkaban.server.external.logviewer.label";
-    
+
     /*
      * Hadoop/Spark user job link.
      * Example:
-     * a) azkaban.server.external.resource_manager_job_url=http://***rm***:8088/cluster/app/application_${application.id}
-     * b) azkaban.server.external.history_server_job_url=http://***jh***:19888/jobhistory/job/job_${application.id}
-     * c) azkaban.server.external.spark_history_server_job_url=http://***sh***:18080/history/application_${application.id}/1/jobs
+     * a) azkaban.server.external.resource_manager_job_url=http://resource-manager-http-address:port/cluster/app/application_${application.id}
```

Comment:
Why not use a more realistic server address e.g. 
https://rm.example.com:8443/....

Reply:
I just followed the example from the apache website: https://hadoop.apache.org/docs/current/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/HistoryServerRest.html
Also, "rm" is not clear enough to users maybe so I choose the full name.

Reply:
I see. The key name has the full name already. So I am not too worried about using the short name in the hostname. How often do you see the full names used in actual server names?

I've seen more example urls using more realistic host names though.

It's a small detail. I will defer to your judgement.

Reply:
I'm ok with your suggestions.